easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.2.20-GCC-6.4.0-2.28.eb

```diff
@@ -0,0 +1,51 @@
+easyblock = 'ConfigureMake'
+
+name = 'OpenBLAS'
+version = '0.2.20'
+
+homepage = 'http://xianyi.github.com/OpenBLAS/'
+description = "OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version."
+
+toolchain = {'name': 'GCC', 'version': '6.4.0-2.28'}
+
+large_src = 'large.tgz'
+timing_src = 'timing.tgz'
+
+source_urls = [
+    # order matters, trying to download the large.tgz/timing.tgz LAPACK tarballs from GitHub causes trouble
+    'http://www.netlib.org/lapack/timing/',
+    'https://github.com/xianyi/OpenBLAS/archive/',
+]
+sources = ['v%(version)s.tar.gz']
+checksums = [
+    '5ef38b15d9c652985774869efd548b8e3e972e1e99475c673b25537ed7bcf394',  # v0.2.20.tar.gz (OpenBLAS)
+    'f328d88b7fa97722f271d7d0cfea1c220e0f8e5ed5ff01d8ef1eb51d6f4243a1',  # large.tgz
+    '999c65f8ea8bd4eac7f1c7f3463d4946917afd20a997807300fe35d70122f3af',  # timing.tgz
+    # OpenBLAS-0.2.20_fix-Intel-L1-cache-size-detection.patch
+    '1d043e4838ec1f90b2b49318b780e3ab13b46133cb72a8d83eb0e3b1b056c4d6',
+    '1e6a046ab658c6e0b351de901d2812db28c2042f9f141416144c2faaf71fbb37',  # OpenBLAS-0.2.20_revert-honor-cpuset.patch
+]
+
+patches = [
+    (large_src, '.'),
+    (timing_src, '.'),
+    'OpenBLAS-%(version)s_fix-Intel-L1-cache-size-detection.patch',
+    'OpenBLAS-%(version)s_revert-honor-cpuset.patch',
```

Comment:
I checked with one of the OpenBLAS co-maintainers whether going forward with OpenBLAS 0.2.20 with these two patches on top makes sense w.r.t. stability, and he confirmed it was.

No need to keep stalling this and hope for a 'quick' OpenBLAS 0.2.21 that includes these fixes...